### PR TITLE
Implement unpin function

### LINKED ISSUE #201: Add unpin functionality

### PR DESCRIPTION
**Pull request by KennethTeo2002**
Closes #201

### CODE REVIEW

**Review by ITLimJiaWei — APPROVED** [review]
LGTM with very minor comment adjustment in a equals testing

**Review by Nicholas-Cheng-De-Fei — APPROVED** [review]
LGTM

File: src/main/java/bizbook/logic/commands/UnpinCommand.java
Comment (Nicholas-Cheng-De-Fei) [inline src/main/java/bizbook/logic/commands/UnpinCommand.java:1]: I think need to update the delete command to also unpin if it deletes a contact that is pinned

**Review by Nicholas-Cheng-De-Fei — CHANGES_REQUESTED** [review]

**Review by Nicholas-Cheng-De-Fei — APPROVED** [review]
LGTM

**Pull request merged**